plasma_framework/test/endToEndTests/FastExit.e2e.test.js

```diff
@@ -219,27 +216,144 @@ contract(
+                            await this.liquidity.safeTransferFrom(alice, bob, this.exitId, { from: alice });
+                        });
+                        it('should have Bob as the new owner of the exit token', async () => {
+                            const nftOwner = await this.liquidity.ownerOf(this.exitId);
+                            expect(nftOwner).to.equal(bob);
+                        });
+                        describe('And then someone processes the exits after two weeks', () => {
+                            before(async () => {
+                                await time.increase(time.duration.weeks(2).add(time.duration.seconds(1)));
 
-                            expect(actualLCBalanceAfterProcessExit).to.be.bignumber.equal(expectedLCBalance);
+                                await this.framework.processExits(config.registerKeys.vaultId.eth, ETH, 0, 1);
+                            });
+                            describe('When Alice tries to claim funds back', () => {
+                                it('should not be successful', async () => {
+                                    await expectRevert(
+                                        this.liquidity.getWithdrawal(this.exitId, { from: alice }),
+                                        'Only the NFT owner of the respective exit can get the withdrawal',
+                                    );
+                                });
+                            });
+                            describe('When Bob tries to claim funds back through the NFT', () => {
+                                before(async () => {
+                                    this.bobBalanceBeforeClaiming = new BN(await web3.eth.getBalance(bob));
+                                    const { receipt } = await this.liquidity.getWithdrawal(this.exitId, {
+                                        from: bob,
+                                    });
+                                    this.bobWithdrawalReceipt = receipt;
+                                });
+                                it('should return the amount to Bob', async () => {
+                                    const actualBobBalanceAfterWithdrawal = new BN(await web3.eth.getBalance(bob));
+                                    const expectedBobBalance = this.bobBalanceBeforeClaiming
+                                        .add(new BN(this.transferTxObject.outputs[0].amount))
+                                        .sub(await spentOnGas(this.bobWithdrawalReceipt));
+
+                                    expect(actualBobBalanceAfterWithdrawal).to.be.bignumber.equal(expectedBobBalance);
+                                });
+                                it('should burn the NFT', async () => {
+                                    expect(await this.liquidity.exists(this.exitId)).to.be.false;
+                                });
+                            });
```

Comment:
there is one more very important test:
1. Alice starts an exit `e1`, via LC,  from a double spend. 
2. Bob starts a valid SE `e2`, via LC.
3. Somebody challenges Alice's exit.
4. Somebody processes the exits. -> Money from `e2` are transferred to LC.
5. Alice tries to burn the token -> should not succeed so that Bob's money is safe.

Reply:
@pik694 Did we finalize on the way to detect challenged exits from the LC? :bow: Also, do we want that to be taken care of here, since we put it under Out of Scope here https://github.com/omgnetwork/plasma-contracts/issues/626 ?

Reply:
oh, sorry, I completely forgot about the fact that that task is not finished.